### DISCUSSION TITLE
React 18 (react/react-dom)

### DISCUSSION THREAD

**Discussion by patrickk** [DefinitelyTyped/DefinitelyTyped#59611] [Request a new @types package]
Hi there, really appreciate the community here for providing us with types. 

Now that React 18 is officially released, are there any plans to release types for React 18? 

Thank you.

**Comment by symbiont-matthew-pirocchi** [comment]
FYI I believe this is the (unfinished) PR for React 18: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210
